File: Array/1832-Check-if-the-Sentence-Is-Pangram/1832-Check-if-the-Sentence-Is-Pangram.java
class Solution {
    public boolean checkIfPangram(String sentence) {
        Map<Character, Integer> ans = new HashMap<>();

        for(int i=0;i<sentence.length();i++) {
            if(!ans.containsKey(sentence.charAt(i))){
                ans.put(sentence.charAt(i), 1);
            }
        }
        return (ans.size() == 26) ? true : false;
    }
}

File: Array/1832-Check-if-the-Sentence-Is-Pangram/1832-Check-if-the-Sentence-Is-Pangram.cpp
class Solution {
public:
    bool checkIfPangram(string sentence) {
        map<char, int> ans;

        for(int i=0;i<sentence.size();i++){
            ans[sentence[i]] = 1;
        }

        return (ans.size() == 26) ? true : false;
    }
};
